applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc

```diff
@@ -18,6 +18,14 @@ As a cluster administrator, you can add multiple cluster-scoped and namespace-sc
 
 As a regular user or project member with the appropriate role-based access control (RBAC) permissions, you can add multiple namespace-scoped Helm chart repositories, apart from the default cluster-scoped Helm repository, and display the Helm charts from these repositories in the *Developer Catalog*.
 
+In the *Developer* perspective of the web console, you can use the *Helm* page to: 
+
+* Create Helm Releases and Repositories using the *Create* button.
+
+* Create, update, or delete a Cluster Scoped or Namespace Scoped Helm chart repository. 
```

Comment:
```suggestion
* Create, update, or delete a cluster-scoped or namespace-scoped Helm chart repository. 
```
- Lower case because they aren't proper nouns.
- Hyphenated because they are [compound modifiers](https://www.grammarly.com/blog/hyphen/#:~:text=Generally%2C%20you%20need%20the%20hyphen,This%20wall%20is%20load%20bearing.).

Compound modifier. (2022, June 1). In Wikipedia. https://en.wikipedia.org/wiki/Compound_modifier

Reply:
Fixed. Thanks for letting me know more about the effect of using modifiers in everyday usage. The example in the wiki article is pretty cool.